curriculum/challenges/english/25-front-end-development/quiz-basic-html/66df3b712c41c499e9d31e5b.md

```diff
@@ -894,3 +894,490 @@ The link has been visited by the user.
 #### --answer--
 
 A user is actively clicking on it.
+
+## --quiz--
+
+### --question--
+
+#### --text--
+
+Which of the following will open the link in a new browser tab?
+
+#### --distractors--
+
+```html
+<a target="_parent" href="https://freecatphotoapp.com">cat photos</a>
+```
+
+---
+
+```html
+<a target="_top" href="https://freecatphotoapp.com">cat photos</a>
+```
+
+---
+
+```html
+<a target="_self" href="https://freecatphotoapp.com">cat photos</a>
+```
+
+#### --answer--
+
+```html
+<a target="_blank" href="https://freecatphotoapp.com">cat photos</a>
+```
+
+### --question--
+
+#### --text--
+
+Which of the following is a void element?
+
+#### --distractors--
+
+`div`
+
+---
+
+`section`
+
+---
+
+`p`
+
+#### --answer--
+
+`img`
+
+### --question--
+
+#### --text--
+
+Which of the following is the correct way to use HTML entities?
```

Comment:
![image](https://github.com/user-attachments/assets/b0e2caac-a1b3-4202-afd0-42573e083e31)

We may want to change these to `` ```txt`` so the syntax highlighting doesn't give it away.